packages/sanity/src/core/releases/tool/overview/ReleasesOverview.tsx

```diff
@@ -134,8 +134,7 @@ export function ReleasesOverview() {
         icon={AddIcon}
         disabled={isCreateBundleDialogOpen}
         onClick={() => setIsCreateBundleDialogOpen(true)}
-        padding={2}
-        space={2}
+        style={{padding: 2}}
```

Comment:
Wouldn't `padding={2}` work? 🤔

Reply:
Not with the ui-components button, we don't allow for `padding` to be passed through to the sanity/ui button here `packages/sanity/src/ui-components/button/Button.tsx`

Reply:
I think we don't need either style or padding here, the default is `padding={2}`